.github/workflows/ci.yml

```diff
@@ -62,6 +62,46 @@ jobs:
       - name: build and check generated types
         if: (${{ success() }} || ${{ failure() }}) # ensures this step runs even if previous steps fail
         run: pnpm build && { [ "`git status --porcelain=v1`" == "" ] || (echo "Generated types have changed — please regenerate types locally with `cd packages/svelte && pnpm generate:types` and commit the changes after you have reviewed them"; git diff; exit 1); }
+      - name: bundle
+        run: |
```

Comment:
Files obtained from `package/svelte/packageJson.exports`

Reply:
After discussing this with Paolo we have agreed it only makes sense to show the compiled code of the compiler (built in L64).

The other files are published to npm with no minification and it's responsibility of the users to bundle them accordingly. Svelte has already close-to-zero dependencies, most of them are for the compiler (only `clsx` goes to the client code, which is tree-shaken if not used). We can have a conversation with the maintainers on what would be the best way to keep track of the package or bundle if there's interest in our tool.